### PR TITLE
Update MonsterTown_manage.md

### PR DESCRIPTION
**Pull request by bhattiburhan3**

### CODE REVIEW

File: management/MonsterTown_manage.md [hunk 1]
```diff
@@ -29,8 +29,13 @@
 6. 07/07 - 07/13
 	* Manager: Burhan Bhatti(bub3)
 	* Goals:
-		1. GOAL1
-		...
+		 1. Place NPC trainers scattered throughout the map
```
Comment (nfarnan) [inline management/MonsterTown_manage.md:32]: Normalize indentation (extra space in front of first itme).

**Review by nfarnan — CHANGES_REQUESTED** [review]
The goals seem to only deal with trainers. Have you got the map laid out as specified in your initial goals and completed the required artwork?

File: management/MonsterTown_manage.md (hunk 1 above)
Comment (bhattiburhan3) [reply management/MonsterTown_manage.md:32]: Yeah we have all the required artwork done and the map is also completely done. The only thing from our midterm goals we have left is NPC related items

**Comment by bhattiburhan3** [comment]
I fixed the indentation as well

**Review by nfarnan — APPROVED** [review]

**Pull request merged**